.github/workflows/production.yml

```diff
@@ -19,6 +19,32 @@ jobs:
         with:
           ref: release
 
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4
+        with:
+          node-version: "^20"
+          cache: yarn
+          cache-dependency-path: yarn.lock
+
+      - name: Setup environment
+        run: sudo apt-get install libelf1
+
+      - name: Install frontend dependencies
+        run: yarn install --immutable
+
+      - name: Build frontend
+        run: NODE_ENV=production yarn build
+
+      - name: Upload frontend build to s3
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks
+        env:
+          AWS_S3_BUCKET: ol-mitopen-app-storage-production
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_PROD }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_PROD }}
+          SOURCE_DIR: "frontends/mit-open/build" # optional: defaults to entire repository
+          DEST_DIR: "frontend" # This dir will get cluttered but it is okay for now
```

Comment:
is this intended to be "frontend" or "frontend(s)"? I was also curious what else ends up in here that clutters it

Reply:
Ah ok this looks like the bucket path /frontend